qa/rpc-tests/test_framework/blocktools.py

```diff
@@ -166,3 +167,45 @@ def createrawtransaction(inputs, outputs, outScriptGenerator=p2pkh):
             tx.vout.append(CTxOut(amount * BTC, outScriptGenerator(addr)))
     tx.rehash()
     return hexlify(tx.serialize()).decode("utf-8")
+
+
+def generateTx(node, txBytes, addrs, data=None, split=1, wallet=None):
```

Comment:
look through all py tests and we defined `generateTx` in `excessive.py`, `uahf.py` and `limits.py`. In all of them the method definition is almost identical.

I wonder if we could define this in a common place and import it when needed. 

Not asking to modify all the above mentioned tests in this PR, just to start the process in this and we will take care of adapting the rest of call-sites in a separate PR.